lib/diagnostic.js

```diff
@@ -38,20 +56,36 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
-
+let pet = {
+  name: 'Majin Buu',
+  species: 'dog',
+  breed: 'mutt',
+  vocalization: 'bark',
+    let vocalize = function () {
+      if (excited) {
+        console.log('bark');
+      }
+    }
+};
+// I know there are errors here, but methods are a little confusing to me still
+//
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
+const Pet = function (name, breed, vocalization) {
+  name: 'name',
```

Comment:
The ```species``` attribute is missing and these lines should be
```
this.name = name;
this.breed = breed;
this.vocalization = vocalization;
```
The way that it's written right now looks like a mix of a function expression and an object literal and this will throw an error.